src/components/manage/Sharing/Sharing.jsx

```diff
@@ -460,29 +460,31 @@ class SharingComponent extends Component {
                   />
                 </p>
               </Segment>
-              <Segment className="actions" attached clearing>
-                <Button
-                  basic
-                  primary
-                  floated="right"
-                  type="submit"
-                  aria-label={this.props.intl.formatMessage(messages.save)}
-                  title={this.props.intl.formatMessage(messages.save)}
-                  loading={this.props.updateRequest.loading}
-                  onClick={this.onSubmit}
-                >
-                  <Icon className="circled" name={aheadSVG} size="30px" />
-                </Button>
-                <Button
-                  basic
-                  secondary
-                  aria-label={this.props.intl.formatMessage(messages.cancel)}
-                  title={this.props.intl.formatMessage(messages.cancel)}
-                  floated="right"
-                  onClick={this.onCancel}
-                >
-                  <Icon className="circled" name={clearSVG} size="30px" />
-                </Button>
+              <Segment className="ui" attached clearing>
+                <div className="ui right aligned container">
```

Comment:
This is not how you use SemanticUI. You don't use the semantic classes outside the SemanticUI components (which uses props to handle them). I'd prefer even to use bare CSS to miss how SemanticUI works.